examples/04-Exporting-ranking-models.ipynb

```diff
@@ -70,9 +70,9 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !tree ./workflow
</pre><p>can you remove this line and  add the following instead?</p><pre class="ql-syntax" spellcheck="false">import seedir as sd
sd.seedir('workflow', style='lines', itemlimit=10, depthlimit=3, exclude_folders='.ipynb_checkpoints', sort=True)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>